templates/tailwind.config.js.ejs

```diff
@@ -0,0 +1,8 @@
+// AUTO-GENERATED - File generated by figma2theme <%- version %>, manual edits may be overwritten
```

Comment:
Unclear at the moment why format-on-save is indenting this file, but not the other.